mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -43,16 +60,13 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
     streams = StreamsFile(streams_filename, streamsdir=indir)
 
     # Note: input file, not a mesh file because we need dycore specific fields
-    # such as refBottomDepth and namelist fields such as config_density0 that
-    # are not guaranteed to be in the mesh file.
-    # attempt to find fields in path-local restart file but default to input
-    # file if restart is unavailable.  This is done to make analysis robust
-    # to being run once the run directory is moved to a new machine.
-    try:
-      inputfile = streams.readpath('restart')[0]
-    except ValueError, e:
-      print e, ' Trying to get mesh from input file.'
-      inputfile = streams.readpath('input')[0]
+    # such as refBottomDepth and namelist fields such as config_density0, as
+    # well as simulationStartTime, that are not guaranteed to be in the mesh file.
+    restartFileNames = streams.readpath('restart')
+    if len(restartFileNames) > 0:
+        inputfile = restarFileNames[0]
```

Comment:
I think that is because the error is caught in streams.readpath
Would try: .. except: raise IOError(..) work?